enemies.js

```diff
@@ -4,6 +4,14 @@ const enemySprites = [],
   deadEnemies = [];
 let spawnedEnemies = 0;
 
+const healthColor = {
+  100: "green",
```

Comment:
This solution won't work as there are some towers that deal more than 20 damage and there will be some instances of enemies having more than (and less than) 100 health, we need something with a percentage range of the enemy health that defines their colour.

Reply:
Added function that colors the healthbar based on relative health